app/models/user.rb

```diff
@@ -54,7 +55,9 @@ def randomize_lat_lon
     self.lon = results[:lon]
   end
 
-  def ensure_lat_lon(address)
+  def ensure_lat_lon
+    valid? # trigger validations
```

Comment:
Il y a l'alias `validate` pour déclencher explicitement les validations, et qui fait et retourne exactement la même chose que `valid?`


```suggestion
    validate
```